application/common/models/User.php

```diff
@@ -312,7 +331,7 @@ public static function findIdentity($id)
      * Finds an identity by the given token.
      *
      * @param string $token the token to be looked for
-     * @return User|null the identity object that matches the given token.
+     * @return User|array the identity object that matches the given token.
```

Comment:
I think this was correct before (as `null`). See http://www.yiiframework.com/doc-2.0/yii-db-activequery.html#one()-detail